routes/tags.js

```diff
@@ -1,90 +1,164 @@
+const e = require('express');
 const express = require('express'); // Use for testing
 const router = express.Router();
 const data = require('../data');
 const tagsData = data.tags;
-
+const groupsData = data.groups;
+const listsData = data.lists
 router.get('/', async (req, res) => {
   try {
     const tags = await tagsData.getAll();
-    res.json(tags);
+    const groups = await groupsData.getAll();
+    const lists = await listsData.getAll();
+    console.log(groups);
+    console.log(lists)
```

Comment:
Did you want to keep these console logs in for now?  We will need to take them all out in the end I think but it might be ok for now

Reply:
Yeah I can take them out, I was just using it for testing, good catch

Reply:
Can also be done later but I know that I would forget lol so I put the comment just in case.

Reply:
Thank you! The change will be in my next PR